extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -27,22 +27,22 @@ export class WorkspaceService implements IWorkspaceService {
 	private excludedProjects: string[] | undefined;
 
 	constructor() {
-		this.getProjectsInWorkspace(undefined, true).catch(err => console.error('Error initializing projects in workspace ', err));
-
-		TelemetryReporter.createActionEvent(TelemetryViews.WorkspaceTreePane, TelemetryActions.ProjectsLoaded)
-			.withAdditionalProperties({
-				openProjectCount: this.openedProjects?.length.toString() ?? '0',
-				exludedProjectCount: this.excludedProjects?.length.toString() ?? '0'
-			}).send();
+		Logger.log(`Calling getProjectsInWorkspace() from WorkspaceService constructor`);
```

Comment:
No duration here since `getProjectsInWorkspace()` is the only thing happening here and that measures duration. Only added this to track where this it's getting called from.